conda_smithy/configure_feedstock.py

```diff
@@ -1357,12 +1357,20 @@ def generate_yum_requirements(forge_config, forge_dir):
         yum_build_setup = textwrap.dedent(
             """\
 
-            # Install the yum requirements defined canonically in the
-            # "recipe/yum_requirements.txt" file. After updating that file,
-            # run "conda smithy rerender" and this line will be updated
-            # automatically.
-            /usr/bin/sudo -n yum install -y {}
-
+            (
+              # Due to https://bugzilla.redhat.com/show_bug.cgi?id=1537564 old
+              # versions of rpm are drastically slowed down when the number of
+              # file descriptors is very high. This can be visible during a
+              # `yum install` step of a feedstock build.
+              # => Set a lower limit in a subshell for the `yum install`s only.
```

Comment:
Let's keep this explanation in conda-smithy. No need to propagate it to each and every feedstock.

Reply:
@jakirkham thoughts?